spec/requests/topics_controller_spec.rb

```diff
@@ -1203,6 +1203,17 @@ def topic_user_post_timings_count(user, topic)
           expect(response.parsed_body['basic_topic']).to be_present
         end
 
+        it 'should trigger only the `topic_edited` event' do
+          put "/t/#{topic.slug}/#{topic.id}.json", params: {
+            title: "New topic title"
+          }
+
+          expect(Jobs::EmitWebHookEvent.jobs.length).to eq(1)
+          job_args = Jobs::EmitWebHookEvent.jobs[0]["args"].first
+
+          expect(job_args["event_name"]).to eq("topic_edited")
```

Comment:
I recently added https://github.com/discourse/discourse/blob/master/spec/support/sidekiq_helpers.rb so we don't have to manually fetch the job_args anymore.